vnext/docs/GettingStarted.md

```diff
@@ -1,6 +1,6 @@
-# Getting Started Guide for React Native Windows C++
+# Getting Started Guide for React Native Windows (vnext)
 
-This is a summary of setup steps needed to install and work with React Native for Windows10 (C++). See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for React Native details.
+This is a summary of setup steps needed to install and work with React Native for Windows (vnext). See the [React Native Getting Started Guide](http://facebook.github.io/react-native/docs/getting-started.html) for React Native details and see [Getting Started Guide - current](https://github.com/microsoft/react-native-windows/blob/master/current/docs/GettingStarted.md) for working with the `current` React Native for Windows implementation.
 
 ## System requirements
 * You can run React-Native for Windows10 apps only on Windows 10 devices; sdk version: 10.0.14393.795 or higher.
```

Comment:
Here we mixed the running os and build target os. the correct description should be:
React Native Windows vNext only supports Windows version late than 10.0.16299.0, so it requires your project to have TargetPlatformMinVersion>=10.0.16299.0 when building.

Reply:
Good catch, I missed that instance. Updated now.